test/datalog_tests/simple.dump.expected

```diff
@@ -155,3 +155,34 @@ Rel2:
 Rel2{0,Option1{0,IP4{300},(true, "foo")}}
 Rel3:
 Rel3{0,IP4{100},Option1{0,IP4{300},(true, "foo")}}
+Aggregate1
+Aggregate1{"a",3}
+Aggregate1{"b",3}
+Aggregate2
+Aggregate2{"a",std_Set { x: {"1", "2", "3"} }}
```

Comment:
Can you also parse sets and vectors in an input file?

Reply:
yep, e.g.,

```

start;
insert HostAddresses(0, ["10.10.10.101", "10.10.10.102", "10.10.10.103"]),
insert HostAddresses(0, ["10.10.10.103", "10.10.10.104", "10.10.10.105"]),
insert HostAddresses(1, ["192.168.0.1", "192.168.0.2", "192.168.0.3"]);
commit;
```